app/main/views/frameworks.py

```diff
@@ -1353,27 +1353,31 @@ def sign_framework_agreement(framework_slug):
 
     supplier = data_api_client.get_supplier(current_user.supplier_id)["suppliers"]
     company_details = get_company_details_from_supplier(supplier)
-    form = SignFrameworkAgreementForm()
+    declaration = data_api_client.get_supplier_declaration(current_user.supplier_id, framework_slug).get('declaration')
     framework_pdf_url = content_loader.get_message(framework_slug, 'urls').get('framework_agreement_pdf_url')
-    contract_titles = {
-        'g-cloud-12': 'Framework Agreement'
+    framework_specific_labels = {
+        'g-cloud-12': {'title': 'Sign agreement',
+                       'contract_title': 'Framework Agreement',
+                       'include_govuk_link': False},
+        'digital-outcomes-and-specialists-5': {'title': 'Sign contract',
+                                               'contract_title': 'Framework Award Form',
+                                               'include_govuk_link': True}
     }
-    contract_title = contract_titles.get(framework_slug)
+    contract_title = framework_specific_labels.get(framework_slug).get('contract_title')
+    title = framework_specific_labels.get(framework_slug).get('title')
+    include_govuk_link = framework_specific_labels.get(framework_slug).get('include_govuk_link')
+    framework_govuk_url = framework.get('frameworkAgreementDetails').get('frameworkURL')
+
+    form = SignFrameworkAgreementForm(contract_title)
 
     # TODO: can we derive this metadata programmatically?
     framework_pdf_metadata = {
-        'g-cloud-12': {'file_size': '487KB', 'page_count': 62}
+        'g-cloud-12': {'file_size': '487KB', 'page_count': 62},
+        'digital-outcomes-and-specialists-5': {'file_size': '97KB', 'page_count': 8}
     }
+
     lots = framework['lots']
```

Comment:
It looks as if we use this only once. Does it need to be a variable?

Reply:
updated in f61397b